src/components/biggive-footer/biggive-footer.tsx

```diff
@@ -89,8 +89,14 @@ export class BiggiveFooter {
 
           <div class="row row-bottom">
             <div class="postscript-wrap">
-              <img class="fr-logo" src={getAssetPath('/assets/images/fundraising-regulator.png')} alt="Registered with FUNDRAISING REGULATOR" />
-
+              <a href="https://www.fundraisingregulator.org.uk/validate?registrant=The+Big+Give+Trust " class="fr-digital-badge">
```

Comment:
```suggestion
              <a href="https://www.fundraisingregulator.org.uk/validate?registrant=The+Big+Give+Trust" class="fr-digital-badge">
```
I know the space was in the original but I think it is almost certainly not needed. Possibly better to remove to avoid confusion? (Same in other places if we agree)

Reply:
Thanks removed in 96c8d4d